domains/kan.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "ishigami"
+    },
+    "record": {
+        "A": ["193.123.102.231"]
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.